e2e/_suites/fleet/features/agent_endpoint_integration.feature

```diff
@@ -71,9 +78,69 @@ Examples: Debian
 @deploy-endpoint-then-remove-it-from-policy
 Scenario Outline: Removing Endpoint from Agent policy stops the connected Endpoint
   Given an Endpoint is successfully deployed with a "<os>" Agent using "tar" installer
+    And the agent is listed in Fleet as "online"
+    And the host name is shown in the Administration view in the Security App as "online"
   When the "Endpoint Security" integration is "removed" in the policy
   Then the agent is listed in Fleet as "online"
     But the host name is not shown in the Administration view in the Security App
+    And the "elastic-endpoint" process is in the "stopped" state on the host
```

Comment:
The background processes checks were moved to its own feature file. Could we move this there?

Reply:
sure, thanks for the reminder.  this pr has been open so long I didn't check it against the other new feature files.

Reply:
actually, i replied too quick.  I don't know if it matters as much.  we have separate file for 'background processes' and a separate file for 'endpoint', separate from the rest of the 'main' agent tests - i don't know which subcategory is more appropriate for a categorization.  once we see it passing it is easy to move it if we can decide which helps the clarity for maintenance and issue reporting.

Reply:
I'm on the fence about these, the scenarios could apply to either backend_processes or agent_endpoint since they overlap pretty heavily. Though the ultimate goal is to test that a backend process is in a certain state so I would lean a little more towards the backend_processes suite file.